servers/partner-ci/common.yaml

```diff
@@ -330,6 +417,28 @@
           fi
 
 - builder:
+    name: get-ui-refs
+    builders:
+      - shell: |
+          #!/bin/bash
+          cherrypick(){
+              git_url=$1
```

Comment:
we need a uniform case for getting refs
but your function is unique
need to discuss